README.md

```diff
@@ -21,21 +21,34 @@ The Squad Leader template demonstrates best practices for building handheld mili
 
 To run the app:
 
-- Android 4.0.1 or higher
+- Android 4.0.3 or higher
   - Limited functionality when running on an Android emulator ([more info](#notes-on-running-with-the-android-emulator))
 - In the Android device's settings, if Developer Options are available, ensure that **"Don't keep activites"** or **"Do not keep activities"** is **unchecked**. See the [User Guide](documentation/UserGuide.md#uncheck-dont-keep-activities) for details.
 
 To build the app from source:
 
 - Android SDK
-  - Android API 14 or higher
-- [ArcGIS Runtime SDK 10.2.4 for Android](https://developers.arcgis.com/en/downloads/)
-  - Even if you have installed the Eclipse plugin from the online update site, you'll need to download the SDK to get the native binary files (see [build steps](#building-with-eclipse)).
-- Eclipse 3.6.2 or higher (Eclipse for Android recommended)
-- Android Support Library
+  - Android API 15 or higher
```

Comment:
There are some other/higher APIs mentioned in the build/config files (and there is a viewshed comment that mentions another) so just wanted to confirm this Android API value